package/MDAnalysis/analysis/hole2/hole.py

```diff
@@ -1248,13 +1241,13 @@ def bin_radii(self, frames=None, bins=100, range=None):
 
         Parameters
         ----------
-        frames: int or iterable of ints, optional
+        frames : int or iterable of ints, optional
             Profiles to include by frame. If ``None``, includes
-            all frames. Default: ``None``
-
-        bins: int or iterable of edges, optional
-            If bins is an int, it defines the number of equal-width bins in the given range. If bins is a sequence, it defines a monotonically increasing array of bin edges, including the rightmost edge, allowing for non-uniform bin widths. Default: 100
-
+            all frames.
+        bins : int or iterable of edges, optional
+            If bins is an int, it defines the number of equal-width bins in the given range.
```

Comment:
Just a reminder to deal with the PEP8 issues (see pep8speaks)

Reply:
Thanks. I fixed the lengths